### PR TITLE
enable debug flag

### PR DESCRIPTION
**Pull request by heartlock**
Signed-off-by: heartlock <21521209@zju.edu.cn>
enable crictl debug flag
@feiskyer @xlgao-zju

### CODE REVIEW

File: cmd/crictl/version.go [hunk 1]
```diff
@@ -42,7 +43,10 @@ var runtimeVersionCommand = cli.Command{
...
 // Version sends a VersionRequest to the server, and parses the returned VersionResponse.
 func Version(client pb.RuntimeServiceClient, version string) error {
-	r, err := client.Version(context.Background(), &pb.VersionRequest{Version: version})
+	request := &pb.VersionRequest{Version: version}
+	logrus.Debugf("VersionRequest: %v", request)
+	r, err := client.Version(context.Background(), request)
+	logrus.Debugf("VersionResponse: %v", r)
 	if err != nil {
 		return err
 	}
```
Comment (feiskyer) [inline cmd/crictl/version.go]: Could you remove the following line: `fmt.Println("VersionResponse:")`

File: cmd/crictl/portforward.go [hunk 2]
```diff
@@ -61,9 +62,14 @@ func PortForward(client pb.RuntimeServiceClient, opts portforwardOptions) error
 		return fmt.Errorf("ID cannot be empty")
 
 	}
+	request := &pb.PortForwardRequest{
+		PodSandboxId: opts.id,
```
Comment (feiskyer) [inline cmd/crictl/portforward.go]: Missing Port?

File: cmd/crictl/version.go (hunk 1 above)
Comment (heartlock) [inline cmd/crictl/version.go]: sure

**Comment by heartlock** [comment]
@feiskyer updated

**Comment by heartlock** [comment]
@feiskyer updated

**Comment by feiskyer** [comment]
LGTM

**Pull request merged**